src/index.html

```diff
@@ -4,9 +4,313 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Catalog</h1>
+  <body class="page-content">
+    <header class="header">
+      <a href="#000" class="header__logo"></a>
```

Comment:
interesting href, but it should be at least meaningful

Reply:
> 

Thank you, I didn't mean to hurt you or that you or your colleague did something wrong, I was just talking about next time corrections. Made necessary changes